test/test-lint-rules.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+DIALECTS=($( ls -d test/fixtures/dialects/*/ ))
+for DIALECT in "${DIALECTS[@]}"
+do
+    echo "Testing $DIALECT SQL files lint without critical errors..."
+    OUTPUT=$(sqlfluff lint ${DIALECT})
```

Comment:
Should we (or do we already) check the `sqlfluff` exit code here as well?

Reply:
We can't. There are LOTS of linting errors (fixtures aren't written to follow all the rules). So it mostly exists with exit code 65. But that's not to say it must. A new dialect could be written with one perfect test case which will exit with 0.

So let's leave as unchecked.